src/EditorFeatures/CSharp/MisplacedUsingDirectives/MisplacedUsingDirectivesUtilities.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Copyright(c) Microsoft.All Rights Reserved.Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.AddImports;
+using Microsoft.CodeAnalysis.CodeStyle;
+using Microsoft.CodeAnalysis.CSharp.CodeStyle;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis.Editor.CSharp;
+
+namespace Microsoft.CodeAnalysis.CSharp.MisplacedUsingDirectives
+{
+    internal static class MisplacedUsingsUtilities
+    {
+        private static readonly CodeStyleOption<AddImportPlacement> s_outsidePreferPreservationOption =
+            new CodeStyleOption<AddImportPlacement>(AddImportPlacement.OutsideNamespace, NotificationOption.None);
```

Comment:
this seems weird.  why here and not in our normal CSharpCodeStyleOptions file?

Reply:
Removed